controllers/workspace/util_test.go

```diff
@@ -264,6 +269,20 @@ func generateConfigMap(name string) *corev1.ConfigMap {
 	}
 }
 
+func generateDwoc(name string) *controllerv1alpha1.DevWorkspaceOperatorConfig {
```

Comment:
This function takes the `name` but doesn't actually use it. Also, maybe we should capitalize the abbreviation `Dwoc` -> `DWOC`?

Reply:
I ended up removing this function since there is the `config.SetGlobalConfigForTesting` function